contrib/guix/manifest.scm

```diff
@@ -0,0 +1,155 @@
+(define-module (bitcoin)
+  #:use-module (gnu)
+  #:use-module (gnu packages)
+  #:use-module (gnu packages autotools)
+  #:use-module (gnu packages base)
+  #:use-module (gnu packages bash)
+  #:use-module (gnu packages check)
+  #:use-module (gnu packages commencement)
+  #:use-module (gnu packages compression)
+  #:use-module (gnu packages cross-base)
+  #:use-module (gnu packages file)
+  #:use-module (gnu packages gawk)
+  #:use-module (gnu packages gcc)
+  #:use-module (gnu packages linux)
+  #:use-module (gnu packages perl)
+  #:use-module (gnu packages pkg-config)
+  #:use-module (gnu packages python)
```

Comment:
Fixed in cd3e947f50db7cfe05c05b368c25742193729a62

Also, because we now do `guix pull` with a commit hash instead of a branch, that python version will never change.